Bottom: 5a928250183d59aaf6bca39e1e1e01bc079c7e62
Top:    00d566d549c5e331b585ef045817a031a2a3a14a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 23:14:34 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
